### DISCUSSION TITLE
Can we control smoothwan remotely?

### DISCUSSION THREAD

**Discussion by lkimloong** [SmoothWAN/SmoothWAN#84] [General & Help]
Wish to check that are we able to control smoothwan remotely from other places where the smoothwan are up and running.

**Comment by bt61** [comment]
> Wish to check that are we able to control smoothwan remotely from other places where the smoothwan are up and running.

I do semi-regularly using the Tailscale functionality built in.

**Reply by lkimloong** [reply]
I did try with the tailscale and I get the ip information but I could not log in to the smoothwan webui. It prompt 404 page not found.
Any thing that I have miss out on tailscale configuration? I have follow the steps advise from the smoothwan ui page.

**Reply by bt61** [reply]
Go to: https://login.tailscale.com/admin/machines

Do you see smoothwan there?  connected, expiry disabled, and subnets?  like this:
![Screenshot 2023-01-05 at 1 58 48 PM](https://user-images.githubusercontent.com/66674857/210712246-2dc9245d-625d-4ca8-b3b2-1a2a9f9581e5.png)

**Reply by lkimloong** [reply]
Yes, I able to see my machine is connected as per your screenshot. I also notice the ip address shown from tailscale. When I try to log in using the provided ip I have the error message "page not found"

**Reply by bt61** [reply]
Did you try using your normal IP?  172.17.17.2 for me, or maybe 192.168.1.1 for you.  That's what I use to access my smoothwan router from outside my home, and I can access other devices on my network with their 172.17.17.X address.  Not the IP given by Tailscale.

**Reply by TalalMash** [reply]
Have you also tried using a different client device to rule out Tailscale?

**Reply by lkimloong** [reply]
I accessing smoothwan thru the public ip show from tailscale. I am actually not really know how to use tailscale.

**Reply by lkimloong** [reply]
Just some updates on the tailscale. i manage to run it
